testnets/int3facetestnet/assetlist.json

```diff
@@ -0,0 +1,283 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "int3facetestnet",
+  "assets": [
+    {
+      "description": "Int3face Testnet blockchain native token",
+      "denom_units": [
+        {
+          "denom": "uint3",
+          "exponent": 0
+        },
+        {
+          "denom": "int3",
+          "exponent": 6
+        }
+      ],
+      "base": "uint3",
+      "name": "Int3face Testnet Token",
+      "display": "int3",
+      "symbol": "INT3",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/int3face/images/int3.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/int3face/images/int3.png",
+          "theme": {
+            "primary_color_hex": "#3d3d3d"
+          }
+        }
+      ],
+      "socials": {
+        "website": "https://testnet.app.int3face.zone/",
+        "twitter": "https://x.com/_Bitfrost_"
+      },
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "Testnet Bitcoin on Int3face",
+      "denom_units": [
+        {
+          "denom": "factory/int31zlefkpe3g0vvm9a4h0jf9000lmqutlh99h7fsd/bitcoin-btc",
+          "exponent": 0
+        },
+        {
+          "denom": "btc",
+          "exponent": 8
+        }
+      ],
+      "base": "factory/int31zlefkpe3g0vvm9a4h0jf9000lmqutlh99h7fsd/bitcoin-btc",
+      "name": "Testnet Bitcoin",
+      "display": "btc",
+      "symbol": "BTC",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "bitcoin",
```

Comment:
Unable to bridge between a mainnet and testnet. But this trace may refer to `bitcointestnet` instead